config/environments/development.rb

```diff
@@ -59,3 +59,7 @@
 
   config.hosts << "website" if ENV['EXERCISM_DOCKER']
 end
+
+Rails.application.routes.default_url_options = {
+  host: "http://lvh.me:3020"
```

Comment:
Does this mean that I should be using this URL to load things locally? It doesn't appear to work with the Docker setup. Do I need to setup something (like a hosts file)?

Reply:
Ah - Interesting/odd that it doesn't work with docker. It should just resolve to localhost (https://www.quora.com/What-is-lvh-me).

The reason I/we use it is because you can't do subdomains on localhost: .e.g `api.localhost`, but you can do `api.lvh.me`.